Packs/GoogleChronicleBackstory/Integrations/GoogleSecOps/GoogleSecOps.yml

```diff
@@ -12,12 +12,27 @@ configuration:
   required: true
   type: 9
   section: Connect
+- display: API URL Format
+  additionalinfo: "Select the API URL format to use for API requests. Default value is '<chronicle>.<REGION>.<rep.googleapis.com>'."
+  name: url_format
+  required: false
+  type: 15
```

Comment:
```suggestion
  type: 15
  defaultvalue: <chronicle>.<REGION>.<rep.googleapis.com>
```

Reply:
We don't want user to configure this parameter until they are facing connection issue which we have mentioned in the help guide.